app/views/schools/add_participants/target_school_cohort_not_set.html.erb

```diff
@@ -0,0 +1,32 @@
+<% content_for :title, "You need to setup #{@target_cohort_start_year}/#{@target_cohort_start_year + 1} cohort at your school to transfer #{add_participant_form.full_name.titleize} into it." %>
+
+<% content_for :before_content, govuk_back_link( text: "Back", href: back_link_path) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <span class="govuk-caption-xl"><%= @school.name %></span>
+    <h1 class="govuk-heading-xl">
+      <%= "We need some more information" %>
+    </h1>
+
+    <p class="govuk-body">
+      Before you can register <%= add_participant_form.full_name.titleize %> for ECF-based training at your school,
```

Comment:
I'm not sure we shold be titleizing peoples names here. It might work for some but will break others (McDonald vs Mcdonald)

Reply:
You are right. I copied it from a similar view. There must be loads across the code.
I will change at least this one. Thanks!

Reply:
Ok, I'll have a look for others. Thanks for the heads up.